### DISCUSSION TITLE
为什么从本地目录以秒传方式上传到阿里云盘会临时占用硬盘 上传完成后就恢复原来大小

### DISCUSSION THREAD

**Discussion by Luckypolo2** [AlistGo/alist#3085] [Q&A]
假设现在剩余100g空间 从本地复制2g文件秒传到阿里云盘 运行过程中剩余空间变为98g左右。后台看到复制完成后，空间恢复100g
请问如果要复制的文件大小大于硬盘剩余空间会如何处理呢

**Comment by xhofe** [comment]
需要先上传至后端计算hash。
